### PR TITLE
Prepare v0.2.0 release

### PR DESCRIPTION
**Pull request by dhimmel**
After other open PRs are merged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -183,6 +183,6 @@ git add manubot/__init__.py
 git commit --message="Set __version__ to $TAG"
 git push
 # Create & push tag (assuming upstream is greenelab remote)
-git tag --annotate $TAG --message="Upgrade to $TAG"
+git tag --annotate $TAG --file=$TAG.md
```
Comment (agitter) [inline README.md]: Is this reading from the file `release-notes/$TAG.md`?  Does the `release-notes` directory need to be specified?

File: release-notes/v0.2.0.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# manubot version 0.2.0 release notes
+
+Manubot version 0.2.0 introduces subcommands to the command line interface.
+The previous command `manubot` to process manuscript content for Pandoc is now `manubot build`.
```
Comment (agitter) [inline release-notes/v0.2.0.md]: `manubot process`?

File: release-notes/v0.2.0.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+New functionality has been added via the `manubot cite` subcommand to retrieve bibliographic metadata for citations (see [CH37](https://github.com/greenelab/manubot/issues/37)).
```
Comment (agitter) [inline release-notes/v0.2.0.md]: [GH37]?

**Comment by dhimmel** [comment]
> I had a few questions.

All your corrections are correct!

**Comment by dhimmel** [comment]
@agitter I'd like to get 0.2 released by Monday next week. Going to Montreal to FORCE11 on Tuesday and will present on Manubot.

Would like to get these two PRs merged before the release:

+ https://github.com/greenelab/manubot-rootstock/pull/134
+ https://github.com/greenelab/manubot/pull/58 (using style from https://github.com/greenelab/manubot-rootstock/pull/134)

don't think there should be much to review as they're small, but @agitter can you take a look?

**Comment by agitter** [comment]
@dhimmel I can review in the next day or two.  Could you please add a screenshot or example PDF to demonstrate the new style?  I like having those in the PR as documentation of what changed.

**Pull request merged**